lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1390,6 +1394,41 @@ def _labeled_unit_config(
         return static_config
 
 
+def _dedupe_job_names(jobs):
+    # Convert to a dict with job names as keys
+    # I think this line is O(n^2) but it should be okay given the list sizes
+    jobs_copy = copy.deepcopy(jobs)
+    jobs_dict = {
```

Comment:
Add comment describing the purpose of this block?
"Create a mapping between job_name values and a list of a all the jobs matching that name"?

Reply:
I think I had that but it was in the wrong place. Fixing.